registration/tests/serializers/test_family.py

```diff
@@ -23,41 +28,89 @@ def setUp(self):
             address="1 Django Boulevard",
             preferred_comms="Shark Tune",
         )
-
         self.family_without_children = Family.objects.create(
             parent=self.parent1,
             email="justkeepswimming@ocean.com",
             cell_number="123456789",
             address="42 Wallaby Way",
             preferred_comms="Whale Song",
         )
-
-        self.family_without_parent = Family.objects.create(
+        self.family_status = Family.objects.create(
+            parent=self.parent3,
             email="justkeepswimming@ocean.com",
             cell_number="123456789",
             address="1 Test Ave",
             preferred_comms="Dolphin Whistle",
         )
-
         self.child1 = Student.objects.create(
             first_name="Albus",
             last_name="Whale",
             role=Student.CHILD,
             family=self.family,
         )
-
         self.child2 = Student.objects.create(
             first_name="Lily",
             last_name="Whale",
             role=Student.CHILD,
             family=self.family,
         )
-
         self.child3 = Student.objects.create(
             first_name="Harry",
             last_name="Tuna",
             role=Student.CHILD,
-            family=self.family_without_parent,
+            family=self.family_status,
+        )
+        self.session1 = Session.objects.create(
+            season="Fall",
+            year="2019",
+            start_date=datetime(2019, 9, 20, 20, 8, 7, 127325, tzinfo=pytz.UTC),
+        )
+        self.session2 = Session.objects.create(
+            season="Spring",
+            year="2021",
+            start_date=datetime(2021, 5, 20, 20, 8, 7, 127325, tzinfo=pytz.UTC),
+        )
+        self.facilitator = User.objects.create(email="user@staff.com")
+        self.class1 = Class.objects.create(
+            name="Test Class 1",
+            session_id=self.session1.id,
+            facilitator_id=self.facilitator.id,
+            attendance=[{"date": "2019-11-01", "attendees": [1, 2]}],
```

Comment:
can we remove the facilitators and set `attendance=[]` for classes here, since these values aren't being tested? just so devs updating this test in the future don't have to worry about them